dockerfiles/Dockerfile.source

```diff
@@ -4,17 +4,17 @@
 # --------------------------------------------------------------
 # Dockerfile to run ONNXRuntime with source build for CPU
 
-FROM ubuntu:18.04
+FROM ubuntu:22.04
```

Comment:
Should we updating all 20.04 to 22.04 as well? So we only need to keep one ubuntu pool?

Reply:
They are docker files.  It should fine to run Ubuntu 20.04 docker images on an Ubuntu 22.04 host. It means we can upgrade all our build machines to 22.04 now, which should not affect who only uses dockers for their build. 
While we should update all such docker files to 22.04, but some of the files are not covered by any build pipeline. Changes on them are hard to validate.